src/library/dao/trackdao.cpp

```diff
@@ -916,7 +916,7 @@ TrackPointer TrackDAO::addTracksAddFile(
 
     const TrackId newTrackId = addTracksAddTrack(pTrack, unremove);
     if (!newTrackId.isValid()) {
-        kLogger.warning() << "TrackDAO::addTracksAddTrack:"
+        kLogger.warning() << "addTracksAddTrack:"
```

Comment:
do we need the churn on the debug and warning statements? (is the class context now part of the output so it doesn't need to be in the message?)

Reply:
I understand that's one benefit of the Logger class.
> do we need the churn on the debug and warning statements?

Some warnings may indeed be just debug logs, and vice versa, but tbh I only removed redundancy and don't plan to do other changes here. If you or someone else has concrete proposals for improvement I'm happy to apply code suggestions.